cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -152,11 +152,9 @@ func makeCmd(certSpec *certsphase.KubeadmCert, cfgPath *string, cfg *kubeadmapiv
 func getSANDescription(certSpec *certsphase.KubeadmCert) string {
 	//Defaulted config we will use to get SAN certs
 	defaultConfig := &kubeadmapiv1alpha3.InitConfiguration{
-		ClusterConfiguration: kubeadmapiv1alpha3.ClusterConfiguration{
-			API: kubeadmapiv1alpha3.API{
-				// GetAPIServerAltNames errors without an AdvertiseAddress; this is as good as any.
-				AdvertiseAddress: "127.0.0.1",
-			},
+		APIEndpoint: kubeadmapiv1alpha3.APIEndpoint{
+			// GetAPIServerAltNames errors without an AdvertiseAddress; this is as good as any.
+			AdvertiseAddress: "127.0.0.1",
```

Comment:
@kad ^ FYI

Reply:
@kad @neolit123 this PR doesn't implemented/changed this bit, only refactored the API struct.
If something should be fixed, this should be address in a separated PR (this one is already XXL 😉)